### PR TITLE
Pkgconfig install

### PR DESCRIPTION
**Pull request by jheo4**
This PR is related to the issue #9 

- [x] Add CMake config file to generate .pc file for pkg-config
- [x] Modify CMakeLists.txt with the config file.

After installing the uvgRTP with pkg-config, uvgRTP can be smoothly used by other projects within their build systems with pkg-config. For the uvgRTP itself, the compile can be done via the below command.
```
g++ main.cc `pkg-config --libs --cflags uvgrtp`
```

### CODE REVIEW

**Comment by altonen** [comment]
Hi @jheo4 

I quickly tested this and at least on my machine, `pkg_search_module(CRYPTOPP libcrypto++)` did not find Crypto++. Changing it to `pkg_search_module(CRYPTOPP libcryptopp)` did find the library but then there seems to be an issue with the linker:

```
examples >> g++ configuration.cc `pkg-config --libs --cflags uvgrtp`
/usr/bin/ld: error: /usr/local/include: read: Is a directory
collect2: error: ld returned 1 exit status
```

```
examples >> pkg-config --libs --cflags uvgrtp
-I/usr/local/include /usr/local/include -L/usr/local/lib -luvgrtp -lpthread -L/usr/local/lib\;-lcryptopp
```
Am I using pkg-config incorrectly or what could be the issue here?
Unfortunately right now I cannot dedicate more time for this so I'll get back to you next week.

**Comment by polusto** [comment]
Hi @jheo4! What is the status of this PR? We recently had a [big update](https://github.com/ultravideo/uvgRTP/commit/837b58351ed65f1526c84571315cb640e4642256) to our CMake system. Do these changes satisfy you, or do you still have interest to work on this PR?

**Comment by polusto** [comment]
We have decided to try to get pkg-config working for the next release, since the release is mostly focused on the build system. I will see if this PR could be used.

File: CMakeLists.txt [hunk 1]
```diff
@@ -100,6 +100,23 @@ if (UNIX)
...
             FILES_MATCHING PATTERN "*.hh"
     )
+
+    # Find pkg-config installed in the system
+    find_package(PkgConfig REQUIRED)
+    if(PkgConfig_FOUND)
+        # Check PKG_CONFIG_PATH, if not defined, use /usr/local/lib/pkgconfig
+        if(NOT DEFINED ENV{PKG_CONFIG_PATH})
+            set( PKG_CONFIG_PATH "/usr/local/lib/pkgconfig")
+            message("PKG_CONFIG_PATH is not found. Set it with ${PKG_CONFIG_PATH}")
```
Comment (jrsnen) [inline CMakeLists.txt:110]: PKG_CONFIG_PATH is not set. Setting it to ${PKG_CONFIG_PATH}.

I think this gives a better impression of what is happening.

**Comment by jrsnen** [comment]
For me, this branch just works. I will try to merge it and fix issues afterwards.

**Comment by jheo4** [comment]
@jrsnen Hi Joni. Sorry for the quiet. It has been quite blasting for me. I checked your comments (thanks for that!) and tested the latest commit of uvgRTP on the clean-slate containers of Ubuntu 18.04 and 20.04. Then, I faced an issue and also resolved it.
I just described the details in Issue #77. Could you check that?

**Pull request merged**